frontend/components/ReviewsOverwrite/theme-gmd/Reviews/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { withPageProductId } from '@shopgate-ps/pwa-extension-kit/connectors/';
```

Comment:
please use everywhere https://developer.shopgate.com/references/engage/hooks#usecurrentproduct instead

Reply:
used "withCurrentProduct" instead of "useCurrentProduct"